Admin/BaseMediaAdmin.php

```diff
@@ -106,7 +106,7 @@ public function getNewInstance()
 
         if ($this->hasRequest()) {
             if ($this->getRequest()->isMethod('POST')) {
-                $media->setProviderName($this->getRequest()->get(sprintf('%s[providerName]', $this->getUniqid()), null, true));
+                $media->setProviderName($this->getRequest()->get(sprintf('%s', $this->getUniqid()), null, true)['providerName']);
```

Comment:
As per the "hacking attempt", I think we should guard this with 
```php
if (!$this->getRequest()->has($this->getUniqId())) {
    throw new \LogicException('form not found in the request');
}
```

And this alone would warrant a unit test IMO. What do you think @sonata-project/contributors